### DISCUSSION TITLE
如何查询章节的本子号

### DISCUSSION THREAD

**Discussion by fordeslorn** [hect0x7/JMComic-Crawler-Python#257] [Q&A] [answered]
我想问问如何查看一个含有多个章节的本子的各个章节的本子号，移动端似乎无法看到具体章节的本子号，这导致我无法只对某个章节进行下载。我查看了提供的jmcomic网站的api部分，小白有点不太看得懂qwq，是否有相应的函数或者option插件以及用法呢

**Comment by hect0x7** [comment, marked as answer]
可以参考：https://jmcomic.readthedocs.io/zh-cn/latest/tutorial/0_common_usage/#_3

你要的应该是photo_id吧，比如 18comic.vip/photo/333，这个章节（photo）的id就是333，333就是你要的章节的本子号

```python
# 1. 先获取章节所在的本子
album = client.get_album_detail(666)
# 2. 获取本子的第一章节
photo = album[0]
# 3. 获取章节的id
pid = photo.id
```

**Comment by fordeslorn** [comment]
<img width="999" alt="Snipaste_2024-07-27_23-40-13" src="https://github.com/user-attachments/assets/d26b6aa0-aef0-416b-91f7-6e28a7bd8e45">
不太会弄啊

**Reply by hect0x7** [reply]
Connection was reset 说明你访问不了禁漫，被墙了。你得先解决网络问题，例如用代理或者找找能连上的域名。

**Reply by fordeslorn** [reply]
<img width="715" alt="0" src="https://github.com/user-attachments/assets/5e24b51a-2d55-425d-9c52-d58ca52189e1">
成这样了

**Reply by hect0x7** [reply]
获取成功了，打印一下呀，print(pid)

**Reply by fordeslorn** [reply]
<img width="703" alt="1" src="https://github.com/user-attachments/assets/80ea0759-8438-4f91-9fa6-eee436bac939">

解决了，只要获取到数组album就行了
非常感谢👍
